diff -Nur openssl-0.9.8za/src/Configure ./Configure
--- openssl-0.9.8za/src/Configure	2014-07-09 18:13:12.000000000 -0700
+++ ./Configure	2014-07-09 17:55:16.000000000 -0700
@@ -1837,11 +1837,16 @@
 # create the ms/version32.rc file if needed
 if ($IsMK1MF && ($target !~ /^netware/)) {
 	my ($v1, $v2, $v3, $v4);
-	if ($version_num =~ /(^[0-9a-f]{1})([0-9a-f]{2})([0-9a-f]{2})([0-9a-f]{2})/i) {
-		$v1=hex $1;
-		$v2=hex $2;
-		$v3=hex $3;
-		$v4=hex $4;
+	if (defined($ENV{"RC_ProjectSourceVersion"}) && defined($ENV{"RC_ProjectBuildVersion"})) {
+		# <rdar://problem/6351135> Bump version number for libeay32.dll
+		my @srcVer = split(/\./, $ENV{"RC_ProjectSourceVersion"});
+		$v1 = $srcVer[0] + 0;
+		$v2 = $srcVer[1] + 0;
+		$v3 = $srcVer[2] + 0;
+		$v4 = $ENV{"RC_ProjectBuildVersion"} + 0;
+	} else {
+		# force version to 0 if this isn't a B&I build
+		$v1 = $v2 = $v3 = $v4 = 0;
 	}
 	open (OUT,">ms/version32.rc") || die "Can't open ms/version32.rc";
 	print OUT <<EOF;
