diff -Nur openssl-0.9.8za/src/crypto/rc5/rc5.h ./crypto/rc5/rc5.h
--- openssl-0.9.8za/src/crypto/rc5/rc5.h	2014-06-05 01:09:08.000000000 -0700
+++ ./crypto/rc5/rc5.h	2014-07-09 17:55:16.000000000 -0700
@@ -73,7 +73,11 @@
 #define RC5_DECRYPT	0
 
 /* 32 bit.  For Alpha, things may get weird */
+#ifdef __LP64__
+#define RC5_32_INT unsigned int
+#else
 #define RC5_32_INT unsigned long
+#endif
 
 #define RC5_32_BLOCK		8
 #define RC5_32_KEY_LENGTH	16 /* This is a default, max is 255 */
