diff -Nur openssl-0.9.8za/src/util/mk1mf.pl ./util/mk1mf.pl
--- openssl-0.9.8za/src/util/mk1mf.pl	2014-07-09 18:33:39.000000000 -0700
+++ ./util/mk1mf.pl	2014-07-09 18:33:15.000000000 -0700
@@ -540,7 +540,7 @@
 SRC_D=$src_dir
 
 LINK=$link
-LFLAGS=$lflags
+LFLAGS=$lflags \$(EXTRA_LFLAGS)   # <rdar://problem/7370791> allow LFLAGS to be modified externally
 RSC=$rsc
 FIPSLINK=\$(PERL) util${o}fipslink.pl
 
@@ -583,8 +583,8 @@
 RANLIB=$ranlib
 MKDIR=$mkdir
 MKLIB=$bin_dir$mklib
-MLFLAGS=$mlflags
-ASM=$bin_dir$asm
+MLFLAGS=$mlflags \$(EXTRA_MLFLAGS)    # <rdar://problem/7370791> allow MLFLAGS to be modified externally
+ASM=$bin_dir$asm \$(EXTRA_ASMFLAGS)   # <rdar://problem/7370791> allow ASMFLAGS to be modified externally
 
 # FIPS validated module and support file locations
 
diff -Nur openssl-0.9.8za/src/util/pl/VC-32.pl ./util/pl/VC-32.pl
--- openssl-0.9.8za/src/util/pl/VC-32.pl	2014-06-05 01:09:08.000000000 -0700
+++ ./util/pl/VC-32.pl	2014-07-09 18:33:15.000000000 -0700
@@ -298,7 +298,7 @@
 	$tmp_def='tmp32dll_$(TARGETCPU)';
 	}
 
-$cflags.=" /Fd$out_def";
+$cflags.=' /Fd$(PDB_PATH)';      # <rdar://problem/7370791> allow PDB path to be modified externally
 
 sub do_lib_rule
 	{
